.circleci/config.yml

```
@@ -31,9 +31,27 @@ jobs:
           command: |
             python ./src/sync_dashboards/main.py sync --config lookml_dashboards.yaml
           no_output_timeout: 15m
+  validate-lookml:
+    docker:
+      - image: circleci/node
+    steps:
+      - checkout
+      - run:
+          name: Execute linting
+          command: |
+            npm set prefix ~/.npm
+            PATH="$HOME/.npm/bin:$PATH"
+            PATH="./node_modules/.bin:$PATH"
+            source ~/.bashrc
+            npm install -g @looker/look-at-me-sideways;
+            if lams --reporting=no;
+              then echo "No linting errors found.";
+              else echo "Linting errors found."; exit 1;
+            fi;
```

I believe the trailing semicolons can be omitted.
```suggestion
            if lams --reporting=no
              then echo "No linting errors found."
              else echo "Linting errors found."; exit 1
            fi
```